Add improvements to lab03 (#674)
UPB-PMRust/website: website/lab/03/index.mdx

| `D5` | `PB4`|`TIM3_CH1` |
| `D6` |`PB10`|`TIM2_CH3` |
| `D9` |`PC6`|`TIM3_CH1` |

> | `D10` |`PC9`|`TIM3_CH4` |

Reviewer: ```suggestion
| `D10` |`PC9`|`TIM3_CH4` (Timer 3 PWM Channel 4) |
```

<Tabs>
  <TabItem value="stm32u5" label="STM32 Nucleo-U545RE-Q" default>
    On the STM32, the ADC peripheral does not use a FIFO interrupt like the RP2. Instead, each ADC (ADC1, ADC2, etc.) can be configured directly, and Embassy provides a driver that handles resolution, averaging, and sample time for you. All we need to do is create the ADC driver, configure it, and then bind it to the pin we want to read from.

| Arduino connector | STM32 pin |ADC|
|-----------|----------|----------|

> | `A0` |`PA0` |`ADC1_IN5` |

Reviewer: ```suggestion
| `A0` |`PA0` |`ADC1_IN5` (ADC1 Channel 5) |
```

Embassy will reset all the peripherals when the `main` function exits, that means `PWM` and `ADC` will stop. Make sure the `main` function never exits so you can see how the circuit behaves.
:::

:::info

The LEDs on the lab board are active‑low. This means they turn ON when the pin is driven LOW and turn OFF when the pin is driven HIGH.

> Because of this wiring, the default PWM polarity (`ActiveHigh`) will feel inverted:

Reviewer: ```suggestion
Because of this wiring, using PWM directly will have the inverse effect:
```
